docker/dev/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="Cray Labs"
+ARG DEBIAN_FRONTEND="noninteractive"
+ENV TZ=US/Seattle
+
+#TODO
+# - add openmpi
+# - add tutorials from other sections
+# - add permissions to the craylabs user
+
+RUN useradd --system --create-home --shell /bin/bash -g root -G sudo craylabs
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y build-essential \
+    git gcc make git-lfs wget libopenmpi-dev openmpi-bin \
+    python3-pip python3 python3-dev cmake \
```

Comment:
Do we still need the pip-installed `cmake`? If that was for SmartRedis, then we can remove it and replace it with `apt-get`/`brew`.

This should have been done as part of the ticket in which we removed `pip install cmake` from SmartRedis. We can open another ticket, as it may go out of scope for this one.

Reply:
this isn't pip installed, this is apt

Reply:
Sorry, my bad!